src/hud_elements.cpp

```diff
@@ -104,12 +104,14 @@ void HudElements::gpu_stats(){
     for(auto per_gpu_info: gpu_info){
          if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gpu_stats]){
         ImGui::TableNextRow();
-        const char* gpu_text;
+        
+        std::string gpu_text;
         if (HUDElements.params->gpu_text.empty())
-            gpu_text = "GPU";
+            gpu_text = "GPU " + per_gpu_info.second.deviceName;
```

Comment:
I don't think the entire device name is going to fit on the hud. I also prefer the device name that comes from pci.ids rather than vulkan

Reply:
The entire name does not fit. We could get the device code names (BONAIRE and CAPE VERDE in my case) instead of the full GPU name. The device name is actually coming from the pci.ids. I'm just storing the data from ```get_device_name``` which queries the PCI Ids
![Screenshot_20210526_010124](https://user-images.githubusercontent.com/5274298/119557971-7b991d00-bdbe-11eb-888c-abec2c07c0a2.png)

Reply:
Thinking if it would be better to just name them "GPU0, GPU1" etc for now

Reply:
I've tweaked the display in the new Pull request. I don't have that much experience with git rebases and messed up my commits. I ended up deleting my repository and adding my changes on top of the develop branch of a fresh fork.